node/src/store/indexed_db_store.rs

```diff
@@ -13,14 +13,17 @@ use serde::{Deserialize, Serialize};
 use serde_wasm_bindgen::{from_value, to_value};
 use tokio::sync::Notify;
 
+use crate::store::header_ranges::{HeaderRange, HeaderRanges, HeaderRangesExt};
+use crate::store::utils::{verify_range_contiguous, RangeScanResult};
 use crate::store::{Result, SamplingMetadata, SamplingStatus, Store, StoreError};
 
 /// indexeddb version, needs to be incremented on every schema schange
-const DB_VERSION: u32 = 2;
+const DB_VERSION: u32 = 3;
```

Comment:
This will preserve existing headers - we're using the same db, so we might as well keep them I think. Pruning window is coming soon, should clean up the unneeded historical data.